src/Microsoft.AspNet.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorPage.cs

```diff
@@ -57,21 +57,21 @@ public CompilationErrorPage()
 #line default
 #line hidden
 
-            WriteLiteral("<!DOCTYPE html>\r\n<html>\r\n    <head>\r\n        <meta charset=\"utf-8\" />\r\n        <t" +
-"itle>");
+            WriteLiteral("<!DOCTYPE html>\n<html>\n    <head>\n        <meta charset=\"utf-8\" />\n        <title" +
```

Comment:
Well something happened here. Did we forget to re-generate the code file or did you generate this on a Linuxish box? :smile: